certification/engine/engine.go

```diff
@@ -98,3 +113,25 @@ func AllChecks() []string {
 	}
 	return all
 }
+
+func OperatorPolicy() []string {
```

Comment:
I am intentionally introducing a new concept.  Policy == a collection of checks.  This is why I wanted to change what we were originally calling Policies to checks, so that we could use the term Policy to refer to a group of checks. This will allow Product Managers to define a Policy with Checks underneath it. Then when you use our tool you can pass in a subcommand stating which Policy (Group of checks) you want to get certified for.  Some checks may belong to multiple policies.